README.md

```diff
@@ -3,8 +3,8 @@
 Zest_Display_LCD board support for Zephyr OS.
 
 ## Usage
-This board enables the following sensors:
-- List Zest_Display_LCD sensors here
+The `zephyr,display` label is exposed by the overlay, allowing to use the spi interface with the default 6TRON spi interface (`sixtron_spi`).
```

Comment:
Please pay attention to case: `spi` -> `SPI`.

Reply:
It also applies to the other acronyms below.

Reply:
Resolved in d1fac08bf21b5fb0b8ff9f45d89920c3568fb184